actors/sunbird-lms-mw/actors/user/src/test/java/org/sunbird/user/dao/impl/UserOrgDaoImplTest.java

```diff
@@ -28,12 +30,20 @@
 
   private static CassandraOperationImpl cassandraOperationImpl;
 
-  @Before
-  public void beforeEachTest() {
+  @BeforeClass
+  public static void setUp() {
     PowerMockito.mockStatic(ServiceFactory.class);
     cassandraOperationImpl = mock(CassandraOperationImpl.class);
   }
 
+  @Before
+  public void beforeEachTest() {
+    Response response = new Response();
```

Comment:
Move the Cassandra object creation to this method

Reply:
Test case is failing when we moved.